### ISSUE TITLE
Magento 2.3 not supported.

### ISSUE THREAD

**Issue by LiamKarlMitchell** [yukoff/magento2-sendgrid-smtp#7]
Store -> System -> SendGrid SMTP
Gives an error.

composer require sendgrid/email-delivery-simplified

"[InvalidArgumentException]
  Could not find a matching version of package sendgrid/email-delivery-simplified. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (stable)."

Installed via downloaded zip manually.

When viewing Magento Dashboard | System | SendGrid | Settings

I get the following error.
1 exception(s):

Exception #0 (InvalidArgumentException): The message should be an instance of \Zend_Mail

Suspect in the code that MessageInterface $message may not be an instance of Zend_Mail.
Magento 2.3 uses Zend Framework 2 not ZF1 so need to use Zend\Mail\Message not Zend_Mail

https://zf2.readthedocs.io/en/release-2.2.3/modules/zend.mail.attachments.html

Can you please release an updated version for Magento 2.3

Although not sure if this repo is the same one.. forgive me if not I've already sent a support ticket to sendgrid.

**Comment by jitheesh-db** [comment]
Same issue with Magento 2.2.8. @LiamKarlMitchell  did you find a fix for this issue?

**Comment by sanjeev-bizspice** [comment]
@LiamKarlMitchell any fix for this issue??

**Comment by syedsaleem086** [comment]
Same issue with Magento 2.2.8. @LiamKarlMitchell 
Any updates

**Comment by jwdegeus** [comment]
Same issue with Magento 2.3.1.

**Comment by sanjeev-bizspice** [comment]
yes, you can use mageplaza smtp extension as an alternative to this. It supports sengrid as well and is compitable with magento 2.3.1

**Comment by edurodriguesdias** [comment]
Thanks @sanjeev-bizspice
